src/python/CRABInterface/Regexps.py

```diff
@@ -77,7 +77,7 @@
 RX_SUBRES_TASK = re.compile(r"^allinfo|allusers|summary|search|taskbystatus|getpublishurl|addwarning|addwebdir|addoutputdatasets|webdir|counttasksbystatus|lastfailures|updateschedd|updatepublicationtime$")
 
 #worker workflow
-RX_WORKER_NAME = re.compile(r"^[A-Za-z0-9\-\._]{1,100}$")
+RX_WORKER_NAME = re.compile(r"^[A-Za-z0-9\-\._%]{1,100}$")
```

Comment:
also this and the following chance in FileTransfer.py are possibly good, but likely not related to the commit comment.